file-formats/ntty/type/zif_aff_ntty_v1.intf.abap

```diff
@@ -35,8 +35,8 @@ INTERFACE zif_aff_ntty_v1
       formatting_profile    TYPE ty_profile,
     END OF ty_content_settings,
 
-    "! <p class="shorttext">Notetype</p>
-    "! Notetype
+    "! <p class="shorttext">Note Type</p>
```

Comment:
Afaik the title here does not relate to any UI text, so you might put smth like `ABAP file format for Note Type objects` that acts as the title of the JSON Schema itself.

Reply:
doen ;)